packages/server/src/modules/match/match.resolvers.ts

```diff
@@ -31,7 +31,7 @@ export class MatchResolver {
       throw new Error('Not found!')
     }
 
-    const improvedSchema = game.resultSchema
+    const improvedSchema = game.resultSchema as JsonSchemaObject
```

Comment:
This type is incorrect, it doesn't have `$schema` at its base

Reply:
The type of the class property should be MinimumSchema or something like that probably

Reply:
As we talked, if we just use the metadata/playerResults property we won't need the `$schema` here anylonger.